src/components/ClientQuote/ClientQuote.tsx

```diff
@@ -0,0 +1,41 @@
+import styles from "./ClientQuote.module.scss";
+import finishLine from "../../assets/images/finishLine.jpg";
+import Image from "next/image";
+
+export function ClientQuote() {
+  return (
+    <main className={styles.main}>
+      <div className={styles.clientQuote}>
+        <p>
+          &quot;Sam Pausha Coaching played a pivotal role in my 5k success.
+          Sam&apos;s personalized training and constant encouragement not only
+          got me across the finish line but also instilled a newfound love for
+          running in my life.&quot; -- Kevin H.
+        </p>
+      </div>
+
+      <div className={styles.imageWrapper}>
+        <div className={`${styles.mobileImage} ${styles.image}`}>
```

Comment:
it'd be better to use the [classnames](https://www.npmjs.com/package/classnames) package. then this becomes `className={classNames(styles.mobileImage, styles.image)}`